R/RcppArmadillo.package.skeleton.R

```diff
@@ -37,13 +37,16 @@ RcppArmadillo.package.skeleton <- function(name="anRpackage", list=character(),
     skelFunName <- ifelse(haveKitten, "kitten", "package.skeleton")
     message("\nCalling ", skelFunName, " to create basic package.")
 
+    
+    
+    
     ## first let the traditional version (or the kitten alternate) do its business
     call <- match.call()
     call[[1]] <- skelFunUsed
+    if ("example_code" %in% names(call)){
+        call[["example_code"]] <- NULL	# remove the example_code argument
+    }
```

Comment:
Isn't this the same as removing `example_code` from the arguments? Ie we have two cases:
- not set, moves on
- set, removes it
So now `example_code` is effectively remove.  I don't have a problem with that (I never used it) but this would need a different deprecation process.

Reply:
It is removed from the the call object, but the original argument itself is used later on (line 107 of original file).

Reply:
Indeed -- good call!